client/post-editor/editor-ground-control/index.jsx

```diff
@@ -351,33 +339,34 @@ export default React.createClass( {
 							{ this.translate( 'Save' ) }
 						</button>
 					}
-					{ this.props.isSaving &&
-						<span className="editor-ground-control__saving">
-							{ this.translate( 'Saving…' ) }
+					{ ! this.isSaveEnabled() &&
+						<span className="editor-ground-control__save-status">
+							{ this.getSaveStatusLabel() }
 						</span>
 					}
 				</div>
-				{
-					this.state.showAdvanceStatus &&
-						<EditPostStatus
-							savedPost={ this.props.savedPost }
-							postDate={ postDate }
-							type={ this.props.type }
-							onSave={ this.props.onSave }
-							onTrashingPost={ this.props.onTrashingPost }
-							onDateChange={ this.setPostDate }
-							site={ this.props.site }>
-						</EditPostStatus>
-				}
 				<div className="editor-ground-control__action-buttons">
-					<button
-						className="editor-ground-control__preview-button button"
+					<Button
+						borderless
+						className="editor-ground-control__preview-button"
 						disabled={ ! this.isPreviewEnabled() }
 						onClick={ this.onPreviewButtonClick }
 						tabIndex={ 4 }
 					>
-						{ this.getPreviewLabel() }
-					</button>
+						<Gridicon icon="visible" /> <span className="editor-ground-control__button-label">{ this.getPreviewLabel() }</span>
+					</Button>
+					<Button
+						borderless
+						className="editor-ground-control__toggle-sidebar"
+						onClick={ this.props.toggleSidebar }
+					>
+						<Gridicon icon="cog" /> <span className="editor-ground-control__button-label">
+							{ this.translate( '{{type/}} Settings', {
```

Comment:
One of the reasons we have this huge list of labels for each post type is that in many languages the sentence structure may be different depending on various features of he post type name (usually gender). This is not a theoretical problem, but a very real one – in core we had to switch approaches long time ago.

Here are two ideas:

* Add one more item to the long list of type-specific strings.
* Or get creative and replace the real sentence with a simpler grammar structure, where the rest doesn’t need to agree with the post type. Options: `{{type/}}: Settings`, `Settings: {{type/}}`, or `Settings for post type “{{type/}}”`.